opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/aggregation.py

```diff
@@ -63,6 +66,18 @@ def collect(self) -> Optional[_PointVarT]:
         pass
 
 
+class AggregationTemporality(IntEnum):
```

Comment:
What's the reason to move this here? `AggregationTemporality` is defined in OTLP, so imo it's part of the exporter options. Users only use it to set the temporality of their exporter/reader right?

Reply:
I agree, the reason to move it is to bypass circular imports. But that does not affect the users, they can import like this: [`from opentelemetry._metrics.export import AggregationTemporality`](https://github.com/open-telemetry/opentelemetry-python/pull/2670/files#diff-998384013178e1a0f05b2e0b4b7968efce3eb5a1bf2267e26b438e04bdcac6b2R53).